beacon-chain/core/transition/state.go

```diff
@@ -237,7 +237,18 @@ func EmptyGenesisState() (state.BeaconState, error) {
 		HistoricalRoots:           [][]byte{},
 		CurrentEpochAttestations:  []*ethpb.PendingAttestation{},
 		PreviousEpochAttestations: []*ethpb.PendingAttestation{},
-
+		PreviousJustifiedCheckpoint: &ethpb.Checkpoint{
+			Epoch: 0,
+			Root:  params.BeaconConfig().ZeroHash[:],
+		},
+		CurrentJustifiedCheckpoint: &ethpb.Checkpoint{
+			Epoch: 0,
+			Root:  params.BeaconConfig().ZeroHash[:],
+		},
+		FinalizedCheckpoint: &ethpb.Checkpoint{
+			Epoch: 0,
+			Root:  params.BeaconConfig().ZeroHash[:],
+		},
```

Comment:
I stopped using this function in favor of the `.empty` method on the `premineGenesisConfig` type, so I can revert this change, but it is probably a good idea to initialize these empty values.